### PR TITLE
WhiteList -> Approved List Language Update

### PR DESCRIPTION
**Pull request by amolsontakke3576**
Screenshot from 2020-04-16 08-49-17
Screenshot from 2020-04-16 08-48-00

Signed-off-by: Amol Sontakke <amols@proximabiz.com>

### CODE REVIEW

File: ccla-managers-and-ccla-signatories/whitelist-contributors.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# Whitelist Contributors
+# Approved List Contributors
 
-Whitelists are lists of domain names, email addresses of individuals, GitHub usernames, or GitHub organization names who are authorized to contribute under a signed Corporate CLA. As a CCLA manager, you allow contributions to projects on behalf of your company by using any whitelist:
+Whitelists are lists of domain names, email addresses of individuals, GitHub usernames, or GitHub organization names who are authorized to contribute under a signed Corporate CLA. As a CCLA manager, you allow contributions to projects on behalf of your company by using any Approved List:
```
Comment (dealako) [inline ccla-managers-and-ccla-signatories/whitelist-contributors.md:3]: The Approved List are lists of domain names...

File: ccla-managers-and-ccla-signatories/whitelist-contributors.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+* Email Approved List allows entities to contribute under an individual email address.
+* GitHub Approved List allows entities to contribute under a GitHub username.
+* GitHub Organization Approved List allows entities to contribute under a GitHub organization name.
 
-Each whitelist applies to the project for which the company has signed a Corporate CLA. The CLA application checks all the whitelists for allowing contributions to a company project. A contributor only needs to be on one whitelist. Contributors can use EasyCLA to send email requests to be associated \(whitelisted\) with the company.
+Each Approved List applies to the project for which the company has signed a Corporate CLA. The CLA application checks all the whitelists for allowing contributions to a company project. A contributor only needs to be on one Approved List. Contributors can use EasyCLA to send email requests to be associated \(whitelisted\) with the company.
 
-_Multiple CCLA managers cannot whitelist the same domain and sign a CCLA for the same company._
+_Multiple CCLA managers cannot Approved List the same domain and sign a CCLA for the same company._
```
Comment (dealako) [inline ccla-managers-and-ccla-signatories/whitelist-contributors.md:12]: _Multiple CCLA managers cannot add to the Approved List the same domain and...

File: ccla-managers-and-ccla-signatories/whitelist-contributors.md (hunk 1 above)
Comment (amolsontakke3576) [reply ccla-managers-and-ccla-signatories/whitelist-contributors.md:3]: Sorry but I don't get it what is the change?

Comment (dealako) [reply ccla-managers-and-ccla-signatories/whitelist-contributors.md:3]: The first word in this line is whitelist,...you missed it

Comment (amolsontakke3576) [reply ccla-managers-and-ccla-signatories/whitelist-contributors.md]: Solved

File: ccla-managers-and-ccla-signatories/whitelist-contributors.md (hunk 2 above)
Comment (amolsontakke3576) [reply ccla-managers-and-ccla-signatories/whitelist-contributors.md]: Solved

**Pull request merged**